platform/security/cas/security-cas-client/src/main/resources/OSGI-INF/blueprint/casfilters.xml

```diff
@@ -12,19 +12,21 @@
  *
  **/
 -->
-
 <blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
-           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0">
+           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0"
+           xmlns:jaxrs="http://cxf.apache.org/blueprint/jaxrs">
 
     <ext:property-placeholder/>
 
     <bean id="casAuthenticationFilter"
           class="org.jasig.cas.client.authentication.AuthenticationFilter">
         <cm:managed-properties persistent-id="ddf.security.cas"
                                update-strategy="container-managed"/>
-        <property name="serverName" value="${org.codice.ddf.system.protocol}${org.codice.ddf.system.hostname}:${org.codice.ddf.system.port}"/>
-        <property name="casServerLoginUrl" value="${org.codice.ddf.system.protocol}${org.codice.ddf.system.hostname}:8443/cas/login"/>
+        <property name="serverName"
+                  value="${org.codice.ddf.system.protocol}${org.codice.ddf.system.hostname}:${org.codice.ddf.system.port}"/>
+        <property name="casServerLoginUrl"
+                  value="${org.codice.ddf.system.protocol}${org.codice.ddf.system.hostname}:8443/cas/login"/>
```

Comment:
✏️ Shouldn't we be using the system property for the port number here and everywhere else?

Reply:
No, this is the CAS port